### ISSUE TITLE
Add the ability to use Bastion services to be able to RDP to cloud instances easily

### ISSUE THREAD

**Issue by JoeBrunsTR** [mRemoteNG/mRemoteNG#2009]
Feature add
Add the ability to use Bastion services to be able to RDP to cloud instances easily

## Expected Behavior

Add the ability to use Bastion services to be able to RDP to cloud instances easily

## Current Behavior

1.77 does not have currently have this

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Operating System and version (e.g. Windows 10 1709 x64):

**Comment by Kvarkas** [comment]
Hi, Bastion is working over https, could you try last NB to test that? I currently don't have setup in azure to test that. But using HTTPs should be no problems.

**Comment by JoeBrunsTR** [comment]
This feature add would be for AWS and GCP as well.

**Comment by cveld** [comment]
@Kvarkas 
I guess Bastion support is not easily provided by just adding a specific http address. 

Is there any progress on this feature request? Makes full sense to add support for Bastion. Not only for rdp, but also for ssh. Please include Entra integration as well when you are on it 😊

From commandline it looks like this:

`az network bastion ssh --ids /subscriptions/<redacted>/resourceGroups/rg-bastion-prd/providers/Microsoft.Network/bastionHosts/bas-prd-001 --target-resource-id /subscriptions/<redacted>/resourceGroups/rg-vm-prd/providers/Microsoft.Compute/virtualMachines/vm-someworkload-prd --auth-type AAD`

**Comment by robertpopa22** [comment]
**Fix available for testing**

A fix for this issue has been implemented in our community fork (`robertpopa22/mRemoteNG`, branch `main`). A beta build including this fix is available from the fork's Releases page.

Please test and report if this resolves your issue.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*

**Comment by JoeBrunsTR** [comment]
I don't think this tool is being maintained anymore sadly.  Try the free version of RDM, https://devolutions.net/remote-desktop-manager/
